server/middleware/security/authentication.js

```diff
@@ -53,7 +53,7 @@ export const parseJwtNoExpiryCheck = (req, res, next) => {
   } else {
     const header = req.headers && req.headers.authorization;
     if (!header) {
-      return next(new Unauthorized('Missing authorization header'));
```

Comment:
if the goal of the middleware is just to authenticate the user, it should never `return next(error);`, otherwise it stops the flow. The controller should simply check for `req.remoteUser`. Or use a `authorization` middleware.